cmd/updatebinary.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"go.keploy.io/server/pkg/service/updateBinary" // Import the updateBinary package
+	"go.uber.org/zap"
+)
+
+// NewCmdUpdateBinary initializes a new command to update the Keploy binary file.
```

Comment:
Please rename the file name to `update` from `updatebinary`. We can also consider the the docker case for update. You can know how you can update the docker image by looking at the alias which we use for docker setup.

Reply:
I assumed that the objective of this PR was strictly to update the binary file that the user has in their system.
CC: @AkashKumar7902

Reply:
Updated the name of the files.

Reply:
Is there a way to find out whether the user is a docker user or is using the binary file which is to be updated? Let me know what would be the correct implementation for the docker users.

Reply:
Understood, Added the message